magwest/templates/preregistration/disclaimers.html

```diff
@@ -2,13 +2,13 @@
 <h4 class="text-danger">Disclaimers</h4>
 <ul>
   <li>This is a one-time non-recurring payment. This is not a subscription service.</li>
-  <li> Please review our badge polices and other <a href="http://magwest.org/faq" target="_faq">frequently asked questions</a> prior to purchasing your badge for important information. </li>
+  <li> Please review our badge polices and other <a href="http://magwest.org/faq" target="_blank">frequently asked questions</a> prior to purchasing your badge for important information. </li>
```

Comment:
Huh... I thought I remembered using different targets was better practice, but googling around doesn't give that indication. Whoops!